libraries/datahandler/src/main/thrift/components.thrift

```diff
@@ -234,6 +234,10 @@ struct Release {
     // information from external data sources
     9: optional  map<string, string> externalIds,
     300: optional map<string, string> additionalData,
+    
+    // Urls for the project
+    70: optional string sourceCodeDownloadurl, // URL for download page for this release source code
+    71: optional string binaryDownloadurl, // URL for download page for this release binaries
```

Comment:
why not use proper camel casing?
`sourceCodeDownloadUrl` and `binaryDownloadUrl`

Reply:
I have just shifted these variables in the .thrift file to fix the **sourceCodeDownloadUrl** column location in the excel. On modifying them to suggested camel case, it affects the generated code that relies on those variable names. For ex. SOURCE_CODE_DOWNLOADURL is after building changes to SOURCE_CODE_DOWNLOAD_URL and hence it gives not found variable compilation error. Then changes have to be made in all places where this has been used.
![Screenshot from 2024-02-29 14-18-30](https://github.com/eclipse-sw360/sw360/assets/142988587/749dab5c-f98e-43e4-b6b9-945d3c791914)

Reply:
Thanks for explaining